account_multicurrency_revaluation_type/model/res_config.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class AccountConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    rate_type = fields.Selection(
+        related='company_id.rate_type',
+        string='Rate type *',
+        help="Revaluations entries will be created "
```

Comment:
IMO this help doesn't make sense here :confused: 
```suggestion
        help="Define default currency rate type to be used for multicurrency revaluation."
```